.github/workflows/clang-format-check.yml

```diff
@@ -0,0 +1,21 @@
+name: clang-format
+
+on: [push]
+
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        path:
+          - 'model'
+          - 'energyplus'
```

Comment:
Would want to add 'fmu' to this, but I didn't feel like going in and Clang Formatting all the FMU code in there or excluding it.